ercc/attestation/ias.go

```diff
@@ -109,6 +111,8 @@ func (ias *intelAttestationServiceImpl) RequestAttestationReport(cert tls.Certif
 		return IASAttestationReport{}, fmt.Errorf("IAS connection error: %s", err)
 	}
 	req.Header.Add("Content-Type", "application/json")
+	req.Header.Add("Ocp-Apim-Subscription-Key", apiKey)
+	logger.Warningf("Sending IAS request %s", req)
```

Comment:
mh ... I have the feeling this should not be a warning. Moreover, this shim logger is kind of broken. Maybe we can omit this logging statement at all. In particular, this has the benefit that we do not add a dependency of the shim.

WDYT?

Reply:
oops, this is a relict of debugging logging. I've actually fixed it but overlooked that file to add for an amend.  Should be fixed now as Debugf